cmd/consul-dataplane/config.go

```diff
@@ -121,6 +121,9 @@ type EnvoyFlags struct {
 	GracefulShutdownPath          *string `json:"gracefulShutdownPath,omitempty"`
 	GracefulPort                  *int    `json:"gracefulPort,omitempty"`
 	DumpEnvoyConfigOnExitEnabled  *bool   `json:"dumpEnvoyConfigOnExitEnabled,omitempty"`
+
+	StartupGracePeriodSeconds *int    `json:"startupGracePeriodSeconds,omitempty"`
+	GracefulStartupPath       *string `json:"gracefulStartupPath,omitempty"`
```

Comment:
I know the other fields in this struct don't have any comments, but I think it would be good to start a trend and comment your fields here.